lib/lev/active_job.rb

```diff
@@ -27,7 +53,20 @@ def perform(*args, &block)
           routine_class = Kernel.const_get(args.pop)
 
           routine_instance = routine_class.new(Lev::BackgroundJob.find!(id))
-          routine_instance.call(*args, &block)
+
+          begin
+            routine_instance.call(*args, &block)
+          rescue StandardError => exception
+            retryable_block = RETRYABLE_EXCEPTION_CONDITIONS[exception.class.name]
+            # Fail the job immediately if it is not retryable
+            unless retryable_block.present? && retryable_block.call(exception)
```

Comment:
Can you collapse these few lines into a private method, e.g. `unless should_retry?(exception)`